cli/src/execute.rs

```diff
@@ -161,6 +160,7 @@ pub async fn execute(
             expiry,
             execution_config,
             callback_config.map(|c| c.into()),
+            None, // A future cli change can implement prover version selection
```

Comment:
The cli does not make a default version line in the sand.   This will get passed on to the contract which will determine the prover version from ProverVersion::default.   

A future cli change can allow the user to choose from different prover versions.